manifests/plugin/puppet.pp

```diff
@@ -0,0 +1,30 @@
+# Pulp puppet plugin
+#
+# @param package_ensure The state or version to install
+# @param proxy_host Host of the proxy server.
+# @param proxy_port Port the proxy is running on.
+# @param proxy_username Proxy username for authentication.
+# @param proxy_password Proxy password for authentication.
+# @param wsgi_processes Number of WSGI processes to spawn for the puppet webapp
+class pulp::plugin::puppet (
+  $package_ensure = undef,
+  # Used in puppet_importer.json.erb
+  Optional[String] $proxy_host = $::pulp::proxy_url,
+  Optional[Integer[1, 65535]] $proxy_port = $::pulp::proxy_port,
+  Optional[String] $proxy_username = $::pulp::proxy_username,
+  Optional[String] $proxy_password = $::pulp::proxy_password,
+  # Used in apache.conf.erb
+  Integer $wsgi_processes = $::pulp::puppet_wsgi_processes,
+) {
+  selboolean { 'pulp_manage_puppet':
+    value      => 'on',
+    persistent => true,
+  }
```

Comment:
This cause an error for me on a proxy install on nightly.  It happened very early in the install, looks to be an ordering issue:

```
[ INFO 2017-05-23 17:27:56 verbose]  Class[Mongodb::Server::Config]: Scheduling refresh of Class[Mongodb::Server::Service]
[ INFO 2017-05-23 17:27:56 verbose]  Class[Mongodb::Server::Service]: Scheduling refresh of Service[mongodb]
[ WARN 2017-05-23 17:28:35 verbose]  /Stage[main]/Mongodb::Server::Service/Service[mongodb]/ensure: ensure changed 'stopped' to 'running'
[ INFO 2017-05-23 17:28:35 verbose]  /Stage[main]/Mongodb::Server::Service/Service[mongodb]: Unscheduling refresh on Service[mongodb]
[ERROR 2017-05-23 17:28:35 verbose]  /Stage[main]/Pulp::Plugin::Puppet/Selboolean[pulp_manage_puppet]: Could not evaluate: Execution of '/usr/sbin/getsebool pulp_manage_puppet' returned 255: Error getting active value for pulp_manage_puppet
```